conftest.py

```
@@ -0,0 +1,6 @@
+def pytest_itemcollected(item):
```

Ruff (0.13.3)

The file is executable but no shebang is present

(EXE002)